playground/stories.js

```diff
@@ -0,0 +1,70 @@
+export * from '../src/components/AccountConnection/README.md';
```

Comment:
Is this list manually generated?

Reply:
It is currently, but I've worked out a way to get this built out automatically, just waiting to demo it to Tim